internals/secrethub/service_gcp_init.go

```diff
@@ -302,6 +304,20 @@ func (l *gcpKMSKeyOptionLister) KeyringOptions() ([]ui.Option, bool, error) {
 	}
 }
 
+func validateGCPKeyring(keyring string) error {
+	if !regexp.MustCompile("^projects/[a-zA-Z0-9-]*/locations/[a-zA-Z0-9-]*/keyRings/[a-zA-Z0-9-_]*$").MatchString(keyring) {
```

Comment:
`*` also matches an empty string.
```suggestion
	if !regexp.MustCompile("^projects/[a-zA-Z0-9-]+/locations/[a-zA-Z0-9-]+/keyRings/[a-zA-Z0-9-_]+$").MatchString(keyring) {
```